src/libraries/System.Runtime/tests/System.Globalization.Extensions.Tests/System.Globalization.Extensions.Tests.csproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
     <TestRuntime>true</TestRuntime>
+    <HybridGlobalization>false</HybridGlobalization>
```

Comment:
Why is it turned off for this test assembly?

Reply:
By default `HybridGlobalization` is enabled. If we want to test `ICU` libs we need to disable `HybridGlobalization`.
I've chosen this test suite (randomly) to test on CI `ICU` libs usage.

Reply:
We have a `HybridGlobalization` job for WASM. Why not to have `ICUGlobalization` for iOS? It's running on request. It's better than testing one lib (always the same) with ICU and all the rest with Hybrid.

Reply:
Sure we can add as a follow up new `ICUGlobalization` job for iOS.

Reply:
But in this PR let's remove this setting to false, in the development process it's fine but I would not merge it.

Reply:
> a follow up new ICUGlobalization job for iOS.

I do not think we need a whole new ICUGlobalization job for iOS. It should be good enough to run the globalization tests with ICU as part of the regular job.

We do use this setup for Hybrid globalization today. There is a variant of globalization tests that run in Hybrid more at https://github.com/dotnet/runtime/blob/main/src/libraries/System.Runtime/tests/System.Globalization.Tests/Hybrid/System.Globalization.IOS.Tests.csproj . 

Now, that the hybrid mode is enabled by default with this PR, this variant of System.Globalization tests is redundant. Should this System.Globalization test variant by switched to test the ICU mode instead?

Reply:
Yes, there are several test suites that are now redundant `System.Runtime.IOS.Tests.csproj`, `System.Globalization.Calendars.IOS.Tests.csproj`, `System.Globalization.Extensions.iOS.Tests.csproj`
 and `System.Globalization.iOS.Tests.csproj`. I will keep `System.Globalization.iOS.Tests.csproj` for ICU flow and delete others.

Reply:
@jkotas "There is a variant of globalization tests that run in Hybrid more at..."
yes, but they also run in non-hybrid mode, [here](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Runtime/tests/System.Globalization.Tests/System.Globalization.Tests.csproj). This way we do not skip testing non-hybrid mode for a certain library and the small subset of hybrid tests is run in the basic azp job. 
However, if we keep the change, then `System.Globalization.Extensions.Tests.csproj` will be never tested for `HybridGlobalization` which is default mode in iOS and we won't detect the issues for it. We can have an additional csproj, same as we did for `Hybrid/System.Globalization.IOS.Tests.csproj` but it's not the same solution as committed now.